client/blocks/reader-post-actions/index.jsx

```diff
@@ -44,24 +48,40 @@ const ReaderPostActions = props => {
 		stats.recordTrackForPost( 'calypso_reader_edit_post_clicked', post );
 	};
 
-	function onPermalinkVisit() {
-		stats.recordPermalinkClick( 'card', post );
-	}
-
 	const listClassnames = classnames( 'reader-post-actions', className );
+	const showLikes = shouldShowLikes( post );
 
 	/* eslint-disable react/jsx-no-target-blank, wpcalypso/jsx-classname-namespace */
 	return (
 		<ul className={ listClassnames }>
-			{ showVisit && (
+			{ showLikes && (
 				<li className="reader-post-actions__item reader-post-actions__visit">
-					<ReaderVisitLink
-						href={ visitUrl || post.URL }
-						iconSize={ iconSize }
-						onClick={ onPermalinkVisit }
-					>
-						{ translate( 'Visit' ) }
-					</ReaderVisitLink>
+					<PostLikesCaterpillar
+						blogId={ +post.site_ID }
+						postId={ +post.ID }
+						gravatarSize={ 12 }
+						className="reader-post-actions__post-likes-caterpillar"
+					/>
+					{ commentCount > 0 && (
+						<div className="reader-post-actions__item-text">
+							{ translate( '%(count)d comment', '%(count)d comments', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **41** times:
`translate( '%d comment', '%d comments' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/post-likers-to-post-card)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).